### PR TITLE
Add "ch" region

### PR DESCRIPTION
**Pull request by phallobst**
Once the PRs from @x-jokay get accepted, the "ch" region will have a number of websites.
How many, you ask? Well... https://www.youtube.com/watch?v=4xgx4k83zzc&t=47

### CODE REVIEW

**Review by Carlgo11 — APPROVED** [review]

**Comment by phallobst** [comment]
Now the Swiss flag is square for the dropdown menu title _and_ item.

File: _includes/navbar.html [hunk 1]
```diff
@@ -17,7 +17,11 @@
             {% assign regions = site.data.regions | sort: "name" %}
             {% assign regions2 = regions | map: 'id' %}
             {% if regions2 contains baseurl %}
-              <span class="flag-icon flag-icon-{{ baseurl }}"></span>
+              {% if baseurl == 'ch' %}
+                <span class="flag-icon flag-icon-squared flag-icon-{{ baseurl }}"></span>
```
Comment (Carlgo11) [inline _includes/navbar.html]: I'd rather see something like this:
```suggestion
                <span class="flag-icon {% if baseurl == 'ch' %}flag-icon-squared{% endif %} flag-icon-{{ baseurl }}"></span>
```

**Comment by phallobst** [comment]
I dropped the hard-coded 'ch' and added a configuration to `regions.yml` - that's much cleaner.

**Review by Carlgo11 — APPROVED** [review]
Looks good! Exciting to see your take on Denmark's flag once they get listed :laughing:

**Comment by phallobst** [comment]
Wait a minute, Denmark is a real country? Mind blown... But I was never good at geometry anyway.

**Comment by jokay** [comment]
denmark 🥳 😄

**Comment by Carlgo11** [comment]
@x-jokay Yes but can you find it on a map?
map

**Review by dsoegijono — APPROVED** [review]

**Pull request merged**